src/styles/_DatePicker.scss

```diff
@@ -52,7 +52,177 @@
+              left: 50%;
+              bottom: 3px;
+              width: 4px;
+              height: 4px;
+              background-color: var(--g-colors-gray-100);
+              border-radius: 50%;
+              transform: translateX(-50%);
+            }
+          }
+
+          div[data-selected] {
+            width: 100%;
+            height: 100%;
+            background-color: var(--g-colors-gray-400);
+            color: var(--g-colors-gray-1000);
+
+            &[data-highlight='black'] {
+              background-color: var(--g-calendarDisplay-backgroundBlack);
+              color: var(--g-calendarDisplay-lightFont);
+              text-decoration: underline 2px solid var(--g-calendarDisplay-lightFont);
+              text-underline-offset: 3px;
+            }
+
+            &[data-highlight='orange'] {
+              background-color: var(--g-calendarDisplay-backgroundOrange);
+              color: var(--g-calendarDisplay-lightFont);
+
+              .date-marker {
+                position: relative;
+                display: inline-block;
+              }
+
+              .date-marker::after {
+                content: '';
+                position: absolute;
+                left: 50%;
+                bottom: 3px;
+                width: 4px;
+                height: 4px;
+                background-color: var(--g-calendarDisplay-lightFont);
+                border-radius: 50%;
+                transform: translateX(-50%);
+              }
+            }
+          }
+
+          div.react-aria-CalendarCell[data-selection-start] {
+            border-top-left-radius: 999px;
+            border-bottom-left-radius: 999px;
+          }
+
+          div.react-aria-CalendarCell[data-selection-end] {
+            border-top-right-radius: 999px;
+            border-bottom-right-radius: 999px;
+          }
+        }
+      }
+    }
+
+    .react-aria-CalendarLegend {
+      width: 16px;
+      height: 16px;
+      border-radius: 2px;
+
+      &[data-highlight='orange'] {
+        background-color: var(--g-calendarDisplay-backgroundOrange);
+        position: relative;
+
+        &::after {
+          content: '';
+          position: absolute;
+          width: 4px;
+          height: 4px;
+          background-color: var(--g-calendarDisplay-lightFont);
+          border-radius: 50%;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+        }
+      }
+
+      &[data-highlight='black'] {
+        background-color: var(--g-calendarDisplay-backgroundBlack);
+        position: relative;
+
+        &::after {
+          content: '';
+          position: absolute;
+          width: 8px;
+          height: 2px;
+          background-color: var(--g-calendarDisplay-lightFont);
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+        }
+      }
+    }
+  }
+
+  // Calendar and Range Calendar commons tyles
+  .react-aria-Calendar,
+  .react-aria-RangeCalendar {
```

Comment:
Nit: i think you could make use of the `toRem` function in this file instead of hardcoding px values

Would def increase the browser default font size and see if the design holds together

Reply:
Done!